### PR TITLE
[BIOX-2875] Enable the Next button in Draw and print

### PR DESCRIPTION
**Pull request by VDykyi**
Added ability to check if scene has content and update if it has changed

### CODE REVIEW

File: src/VAC/Scene.cpp [hunk 1]
```diff
@@ -810,6 +810,17 @@ SurfaceType Scene::getLoadedSurfaceType()
     return surfaceType;
 }
 
+bool Scene::hasSceneContent() const
+{
+    for (auto layer: layers_)
+    {
```
Comment (marcus-sonestedt) [inline src/VAC/Scene.cpp]: run clang-format, or install in qt creator

**Review by marcus-sonestedt — APPROVED** [review]

File: src/VAC/Scene.cpp (hunk 1 above)
Comment (VDykyi) [reply src/VAC/Scene.cpp]: It's VPaint and it doesn't have clang format, I have installed clang plugin....

Comment (VDykyi) [reply src/VAC/Scene.cpp]: Or we need to apply clang format to the VPaint too?

Comment (VDykyi) [reply src/VAC/Scene.cpp]: If I will run clang check on vpaint code - will be a lot of changes...

Comment (marcus-sonestedt) [reply src/VAC/Scene.cpp]: No, I always forget that we don't do that in VPaint.. 

Nevermind... I need more coffee! 😅

Comment (VDykyi) [reply src/VAC/Scene.cpp]: done

**Pull request merged**